src/AutoRest.CSharp/DataPlane/Output/DataPlaneClient.cs

```diff
@@ -1,74 +1,71 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
 using System.Collections.Generic;
 using System.Linq;
+using AutoRest.CSharp.Common.Input;
 using AutoRest.CSharp.Common.Output.Builders;
 using AutoRest.CSharp.Generation.Types;
-using AutoRest.CSharp.Generation.Writers;
-using AutoRest.CSharp.Input;
 using AutoRest.CSharp.Output.Builders;
 using AutoRest.CSharp.Output.Models.Requests;
-using AutoRest.CSharp.Output.Models.Responses;
 using AutoRest.CSharp.Output.Models.Shared;
 using AutoRest.CSharp.Output.Models.Types;
+using AutoRest.CSharp.Utilities;
 
 namespace AutoRest.CSharp.Output.Models
 {
     internal class DataPlaneClient : TypeProvider
     {
-        private readonly OperationGroup _operationGroup;
+        private readonly InputClient _inputClient;
         private readonly BuildContext<DataPlaneOutputLibrary> _context;
```

Comment:
I think our agreement is HLC still take M4 as input. Why change OperationGroup into InputClient?

Reply:
The same input is used in the `DataPlaneRestClient` which utilizes shared `RestClientBuilder`. I'm planning to get rid of HLC as a separate output library soon after DPG gets serialization, so there is no reason to keep `OperationGroup` around: it will make code more complicated without any benefits.